examples/bank/bank.go

```diff
@@ -139,7 +208,11 @@ func (bank *Bank) continuousMoneyTransfer() {
 		from := makeAccountID(rand.Intn(bank.numAccounts))
 		to := makeAccountID(rand.Intn(bank.numAccounts))
 		exchange := rand.Int63n(100)
-		bank.moveMoney(from, to, exchange)
+		if *useTransaction {
+			bank.moveMoneyTransaction(from, to, exchange)
+		} else {
+			bank.moveMoney(from, to, exchange)
+		}
```

Comment:
A bit ugly to duplicate the functionality of `moveMoney` and `moveMoneyTransaction`. I think you can mostly unify the two and doing so will be even easier with the new client API.